tests/profile/profile-page.spec.ts

```diff
@@ -63,7 +62,37 @@ test.describe('Profile Page - Basic display tests', () => {
       await profilePage.validateClickedTabAppeared('web-archive');
       await profilePage.validateResultCountElement('web-archive');
     });
+  
+    await test.step(`Check if profile tabs are displayed`, async () => {
+      await profilePage.validateUnownedProfilePageTabs();
+    });
+  
```

Comment:
I think this step is unnecessary to add here -- seems like it might have just snuck down here from above during the rebase.

Reply:
👍  Agreed; Will remove.

Reply:
Just noting: I'll merge this into my branch as-is, and make this change there.